### PR TITLE
#81 Continue to replace MatchersAssert.assertThat with Assertion

### PR DESCRIPTION
**Pull request by Iprogrammerr**
As described in #81. All occurrences of MatchersAssert.assertThat() were replaced with Assertion in TextIsTest and ThrowsTest. Puzzle was added to continue in ScalarHasValueTest.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#82](https://github.com/llorllale/cactoos-matchers/issues/82) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#82](https://github.com/llorllale/cactoos-matchers/issues/82) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C63314D6Z/a08d3f07-45b9-44e8-ba9a-f715750a1a20); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by fabriciofx** [comment]
@Iprogrammerr please, check Travis fails. Thanks!

**Comment by Iprogrammerr** [comment]
@fabriciofx It fails because there is a failed test in RunsInThreadTest, which was not modified and it passed one build ago, which means that there is a race condition. I do not think that it should be ignored. @llorllale WDYT?

**Comment by fabriciofx** [comment]
@Iprogrammerr right, but it needs investigation. How can I know if Travis fails because yours modifications or others problems? I don't consider see Travis logs a option, sorry. @llorllale Could you check it?

**Comment by Iprogrammerr** [comment]
@fabriciofx Agreed. Let's wait for @llorllale, because he should see this issue as it can be hard to replicate.

**Comment by llorllale** [comment]
@Iprogrammerr wait for #77

**Comment by Iprogrammerr** [comment]
@llorllale Ready.

**Comment by fabriciofx** [comment]
@Iprogrammerr please, check it again, ok?

**Comment by Iprogrammerr** [comment]
@fabriciofx Could you clarify? Build passed and you made no comments regarding my changes.

**Comment by fabriciofx** [comment]
@Iprogrammerr sorry. I forgot to send my review.

File: src/test/java/org/llorllale/cactoos/matchers/ThrowsTest.java [hunk 1]
```diff
@@ -47,30 +46,30 @@
      */
     @Test
     public void matchPositive() {
-        MatcherAssert.assertThat(
-            "The thrown exception is valid.",
+        new Assertion<>(
+            "The thrown exception is not valid.",
             () -> {
```
[the remark this answers is not in the document]
Comment (Iprogrammerr) [reply src/test/java/org/llorllale/cactoos/matchers/ThrowsTest.java]: @fabriciofx They are not. See https://stackoverflow.com/questions/41961392/why-does-a-java-lambda-which-throws-a-runtime-exception-require-brackets

**Comment by Iprogrammerr** [comment]
@fabriciofx Refactored.

File: src/test/java/org/llorllale/cactoos/matchers/ThrowsTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (fabriciofx) [reply src/test/java/org/llorllale/cactoos/matchers/ThrowsTest.java]: @Iprogrammerr I didn't know this detail... thanks for show me!

**Comment by fabriciofx** [comment]
@llorllale this PR seems fine to me. Could you merge?

**Comment by fabriciofx** [comment]
@rultor merge

File: src/test/java/org/llorllale/cactoos/matchers/ThrowsTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Iprogrammerr) [reply src/test/java/org/llorllale/cactoos/matchers/ThrowsTest.java]: @fabriciofx No problem:)

File: src/test/java/org/llorllale/cactoos/matchers/ScalarHasValueTest.java [hunk 2]
```diff
@@ -40,6 +40,9 @@
  * Test case for {@link ScalarHasValue}.
  *
  * @since 1.0
+ * @todo #52:30min Replace all uses of MatcherAssert.assertThat() with
```
[the remark this answers is not in the document]
Comment (Iprogrammerr) [reply src/test/java/org/llorllale/cactoos/matchers/ScalarHasValueTest.java]: @llorllale Fixed.

**Comment by llorllale** [comment]
@rultor merge

**Pull request merged**